src/components/companyOverview.js

```diff
@@ -12,7 +12,7 @@ export const CompanyOverview = (props) => {
   const companyInfo = useSelector(companySelector);
   const ticker = useSelector(tickerSelector);
   return (
-    <DisplayWrapper height={props.height}>
+    <DisplayWrapper {...props}>
```

Comment:
I generalized this approach of passing CSS props from outside the component for a lot of our features. This applies all props given to the CompanyOverview, for example, to the DisplayWrapper inside it, so I didn't need to make any extra DisplayWrappers in the layout.